src/test/scala/org/camunda/feel/impl/builtin/BuiltinTemporalFunctionsTest.scala

```diff
@@ -108,4 +108,9 @@ class BuiltinTemporalFunctionsTest
     eval(""" abs(duration("-P2M")) """) should be(ValYearMonthDuration("P2M"))
   }
 
+
+  it should "return the the last day of the month" in {
+    eval(""" last day of the month(date(2022,10,17)) """) should be(ValDate(LocalDate.parse("2022-10-31")))
```

Comment:
:x: We should also verify that we can invoke the function with a date-time (with/without time zone). Similar to the other test cases using `$date`, `$localDateTime`, and `$dateTime`.